create_database.py

```
@@ -0,0 +1,63 @@
+    print("\nGenerando embeddings con Gemini y guardando en ChromaDB...")
+    print("(Esto puede tardar varios minutos)")
+    
+    count = 0
+    batch_size = 100
+    for i in range(0, len(text_chunks), batch_size):
+        batch_chunks = text_chunks[i:i+batch_size]
+        try:
+            response = genai.embed_content(
+                model="models/text-embedding-004",
+                content=batch_chunks,
+                task_type="RETRIEVAL_DOCUMENT"
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)